src/core/reopt.jl

```diff
@@ -79,8 +122,19 @@ end
 Method for use with Threads when running BAU in parallel with optimal scenario.
 """
 function run_reopt(t::Tuple{JuMP.AbstractModel, AbstractInputs})
-	run_reopt(t[1], t[2]; organize_pvs=false)
-    # must organize_pvs after adding proforma results
+
+	instantiate_logger()
+
+	try
```

Comment:
There's a lot of duplication of this try/catch/handle_errors code that seems like it could be eliminated. For example, this run_reopt function calls a different run_reopt but the error handling is put in both of them.